tests/datamodules/test_spacenet.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we instead test this with a trainer and delete this file?

Reply:
This brought up an issue in plotting. In SpaceNet's plot method we do percentile normalization of the input image (https://github.com/microsoft/torchgeo/blob/main/torchgeo/datasets/spacenet.py#L327). Now, because SpaceNet1 test data is 2 x 2 and I zero-pad to 448 x 448, we get a divide by zero error in the percent normalization method.

Options:
- Make SpaceNet test data larger (this is what I did)
- Add an epsilon value in the denominator of the percentile normalization method
- Switch to a subsistence farming lifestyle and never touch a computer again

Reply:
As long as it isn't possible to hit this zero division error when using the real data, I'm fine with either 1 or 3. 2 might be a good idea though regardless. Up to you.

Reply:
I chose 2